packages/shared/src/types/volunteer.ts

```diff
@@ -13,6 +13,7 @@ export interface IVolunteer {
     lastName: string;
     gender: string;
     birthDate: string;
+    age: number;
```

Comment:
Let's change it to birth year, please. If we keep age in DB, we'll need to recalculate it

Reply:
We have decided to set an age property due to some errors during the filter with the birth year.